saltcellar/api/status/view.py

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""View for server status."""
+
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework import permissions
+from rest_framework.response import Response
+
+from api import API_VERSION
+from api.status.model import ServerInformation
+
+from saltcellar.environment import (commit,
+                                    modules,
+                                    platform_info,
+                                    python_version)
+
+
+@api_view(['GET'])
```

Comment:
Have you looked at class based views in DRF? There's a series of mixins that cover standard CRUD which makes writing APIs where the operations map 1:1 with your DB model ⚡️ fast. Since this does not do that we'd still be writing a custom retrieve/get method, so this is fine for now. I'll just note that we may want to use them in the future and for consistency we could refactor this to use class based views then.